Paladin.cpp

```diff
@@ -1,7 +1,28 @@
 #include "Paladin.h"
+#include "Utility.h"
+#include <string>
 
-// Paladin::Paladin
+Paladin::Paladin(std::string nameGiven, int hitPoints, int armor ) : Character(hitPoints, armor, 10), name(nameGiven)
+{
+    helpfulItems = makeHelpfulItems(2);
+    defensiveItems = makeDefensiveItems(6);
+}
+Paladin::~Paladin()
+{
+    
+}
 
-// Paladin::getName
+const std::string& Paladin::getName()
+{
+    return name;
+}
+std::string Paladin::getStats()
+{
+    return getCharacterStats(this);
+}
+void Paladin::attack(Character& other)
+{
+    std::cout << name << " is attacking " << other.getName() << "!!!" << std::endl;
+    Character::attack(other);    
+}
```

Comment:
This override does nothing new other than add some exclamation marks.
- [ ] remove